Sources/Model/TransferMethod/HyperwalletBankAccount.swift

```diff
@@ -24,73 +24,66 @@ public final class HyperwalletBankAccount: HyperwalletTransferMethod {
     override private init(data: [String: AnyCodable]) {
         super.init(data: data)
     }
-
+    /// Initialization of HyperwalletBankAccount
```

Comment:
update to - /// The required initializer

Reply:
Updated